### DISCUSSION TITLE
Solid.js on older browsers versions

### DISCUSSION THREAD

**Discussion by johneisenheim** [solidjs/solid#1162] [Q&A]
Hello everyone. First of all, solid.js is great! 🎉
I'm wondering if it is possible to use it on older Chrome versions, like for example >= 40. 

Thank you all,

**Comment by atk** [comment]
Proxy is only supported after Chrome 48, so you would have to rely on transpilation and emulation of those features not supported by the browser. While it could work, there is no officially support from solid-js of such a solution, as it would dramatically increase testing requirements.
